docs/src/rules/require-unicode-regexp.md

```diff
@@ -7,7 +7,7 @@ further_reading:
 ---
 
 
-RegExp `u` flag has two effects:
+`RegExp` `u` flag has two effects:
```

Comment:
```suggestion
RegExp `u` flag has two effects:
```

I think we should keep this without backticks as RegExp here refers to "regular expressions" rather than the `RegExp` constructor.